lib/core/datatable/components/datatable/datatable.component.scss

```diff
@@ -315,66 +330,6 @@ $data-table-cell-min-width-file-size: $data-table-cell-min-width !default;
         }
     }
 
-    .adf-datatable-cell-header {
```

Comment:
While dragging header column, column is taken out from the context of data-table therefore moved this class outside of adf-datatable-list